.github/workflows/psp-ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI PSP
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [run_build]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pspdev/pspdev:latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Prepare Distribution
+      run: |
+        ./configure --host=psp --disable-debug --enable-plugins --default-dynamic --enable-release
+        make distclean
+
+    - name: Compile ScummVM
+      run: |
+        ./configure --host=psp --disable-debug --enable-plugins --default-dynamic --enable-release
+        make -j$(getconf _NPROCESSORS_ONLN)
+
+    - name: Prepare Release
+      run: |
+        export VERSION=`make print-distversion`
+        export DISTS=`make print-dists`
+        mkdir scummvm-$VERSION
+        cp -r $DISTS EBOOT.PBP plugins scummvm-$VERSION
+        mkdir scummvm-$VERSION/kbd
```

Comment:
Here, kbd.zip should be copied instead of creating a kbd folder and copying the single .png files.